ci/test_complete.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 
+##
 # Compares options in `rg --help` output to options in zsh completion function
+#
+# @todo If we could rely on zsh being installed we could change all of this to
+# simply source the completion-function file and pull the rg_args array out...
```

Comment:
Oh, i wasn't sure it was an option. Knowing that it is, i'll probably do that the next time i need to make changes to the function.